smartsim/_core/_install/buildenv.py

```diff
@@ -126,6 +129,8 @@ class RedisAIVersion(Version_):
             "onnxmltools": "1.10.0",
             "scikit-learn": "1.0.2",
             "torch": "1.7.1",
+            "torch_cpu_suffix": "+cpu",
+            "torch_gpu_suffix": "+cu110",
```

Comment:
If this is a specific (NVIDIA) GPU, please document that here

Reply:
Good call. I have renamed this to `torch_cuda_suffix` to allow for the possibility for us to specify an alternate library (e.g. ROCm or another accelerator library)